common/lib/xmodule/xmodule/templates/problem/string_response.yaml

```diff
@@ -2,38 +2,45 @@
 metadata:
     display_name: Text Input
     markdown: |
-        A text input problem accepts a line of text from the student, and evaluates the input for correctness based on an expected answer.
-    
-        The answer is correct if it matches every character of the expected answer. This can be a problem with international spelling, dates, or anything where the format of the answer is not clear.
-    
-        >>Which US state has Lansing as its capital?<<
+        In fill-in-the-blank problems, also known as "text input" problems, learners enter text into a response field. The text can include letters and characters such as punctuation marks. The text that the learner enters must match your specified answer text exactly.
+        To create a fill-in-the-blank problem in the simple editor:
+        1. Enter the text of the main question between two pairs of angle brackets that point inward (>><<).
+        2. On a new line, enter an equal sign (=) followed by a space and the text of the correct answer.
+        3. (optional) To specify more than one answer, on a new line, enter "or=" followed by a space and the text of the answer. Repeat this step for each additional correct answer.
```

Comment:
Do we want to highlight that you might want to specify additional correct answers so that student answers that use different case, or are spelled slightly differently, can also be counted as correct?